components/mpas-ocean/src/shared/mpas_ocn_vel_self_attraction_loading.F

```diff
@@ -643,11 +649,11 @@ subroutine ocn_vel_self_attraction_loading_init(domain,err)!{{{
                 gridToMpasFile = trim(config_grid_to_mpas_weights_file)
 
                 ! Open netcdf weights files
-                call check( nf90_open(path = mpasToGridFile, mode = nf90_nowrite, ncid = toNcId) ,err)
-                call check( nf90_open(path = gridToMpasFile, mode = nf90_nowrite, ncid = fromNcId) ,err)
+                call check( nf90_open(path = mpasToGridFile, mode = nf90_nowrite, ncid = toNcId) ,err_tmp)
+                call check( nf90_open(path = gridToMpasFile, mode = nf90_nowrite, ncid = fromNcId) ,err_tmp)
 
                 ! Get dimension ID
-                call check( nf90_inq_dimid(toNcId, "n_s", toNsDimId) ,err)
+                call check( nf90_inq_dimid(toNcId, "n_s", toNsDimId) ,err_tmp)
```

Comment:
@knbarton, should these be `err` since the `ior` with the temporary error variable is done in the `check` subroutine?